pages/sign-up/index.js

```diff
@@ -37,7 +37,7 @@ const SignUp = ({ organisations, error }) => {
     setOrganisation(selectedOrganisation);
   };
 
-  const verifyEmail = (email) => email.match(/@nhs\.co\.uk/);
+  const verifyEmail = (email) => email.match(/@madetech\.com/);
```

Comment:
Would it be possible for this to be configurable from the start?
Hard coding like this is going to lead to issues further down the line when we inevitably need to change it.

Reply:
Sorry, I accidentally changed that line. It was previously hardcoded to nhs.co.uk. I am going to revert it to the old code now.